server/app/com/xsn/explorer/gcs/GolombEncoding.scala

```diff
@@ -185,7 +189,15 @@ object GolombEncoding {
   def encode(block: Block.HasTransactions): Option[GolombCodedSet] = {
     val key = SipHashKey.fromBtcutil(block.hash)
     val encoder = default(key)
-    val addresses = block.collectAddresses
-    encoder.encode(addresses.map(_.string))
+
+    val pubKeyScripts = block.transactions.flatMap(transaction => {
+      val inputScripts = transaction.inputs.filter(_.script.isDefined).map(_.script.get)
```

Comment:
Use `collect` instead of `filter` and `map`, intellij should have suggested this.